### PR TITLE
Make port in swagger docs configurable

### PR DESCRIPTION
**Pull request by deo002**
This PR makes port in swagger docs configurable. Fetches the env variable "PORT" and assigns it to the swagger config variable "Host" on start.

### CODE REVIEW

File: pkg/api/api.go [hunk 1]
```diff
@@ -33,14 +36,23 @@ import (
...
+
+const DEFAULT_PORT = 8080
+
 func Router() *gin.Engine {
+
+	if os.Getenv("PORT") != "" {
+		docs.SwaggerInfo.Host = fmt.Sprintf("localhost:%s", os.Getenv("PORT"))
+	} else {
+		docs.SwaggerInfo.Host = fmt.Sprintf("localhost:%d", DEFAULT_PORT)
+	}
```
Comment (avinal) [inline pkg/api/api.go:54]: Seems fine, a better way would be to remove the else completely. And making this logic concise. 

```suggestion
  port := os.Getenv("PORT")
	if len(port) == 0 {
	  port = DEFAULT_PORT
	}
  docs.SwaggerInfo.Host = fmt.Sprintf("localhost:%s", port)
```

File: pkg/api/api.go [hunk 2]
```diff
@@ -33,14 +36,23 @@ import (
...
 //
-//	@host						localhost:8080
 //	@BasePath					/api/v1
 //
 //	@securityDefinitions.apikey	ApiKeyAuth
 //	@in							header
 //	@name						Authorization
 //	@description				Token from /login endpoint
+
+const DEFAULT_PORT = 8080
```
Comment (avinal) [inline pkg/api/api.go:46]:
```suggestion
const DEFAULT_PORT = "8080"
```

**Review by avinal — CHANGES_REQUESTED** [review]
Few suggestions

**Review by avinal — APPROVED** [review]
Please update the PR description and it should be good to be merged.

**Review by GMishx — APPROVED** [review]
Tested, working as expected.

**Pull request merged**